sandersaares.devops-scripts/Set-VersionStringBranchPrefix.ps1

```diff
@@ -66,6 +66,8 @@ function Set-VersionStringBranchPrefix {
             # What? Not sure - should have been a reference with the /s in it but okay, just go with it.
             $versionPrefix = $sourceRef
         }
+
+        $versionPrefix = $versionPrefix.Replace("_", "-");
```

Comment:
Would also be nice to leave a comment here explaining why this sort of sanitization is needed.